hw/top_earlgrey/ip/xbar_main/dv/autogen/xbar_cov_excl.el

```diff
@@ -0,0 +1,494 @@
+Toggle tl_keymgr_o.a_address [21]
+Toggle tl_keymgr_o.a_address [22]
+Toggle tl_keymgr_o.a_address [23]
+Toggle tl_keymgr_o.a_address [24]
+Toggle tl_keymgr_o.a_address [25]
+Toggle tl_keymgr_o.a_address [26]
+Toggle tl_keymgr_o.a_address [27]
+Toggle tl_keymgr_o.a_address [28]
+Toggle tl_keymgr_o.a_address [29]
+Toggle tl_keymgr_o.a_address [31]
+
+Toggle tl_corei_i.a_source [7]
+Toggle tl_corei_o.d_source [7]
+Toggle tl_cored_i.a_source [7]
+Toggle tl_cored_o.d_source [7]
+Toggle tl_dm_sba_i.a_source [7]
+Toggle tl_dm_sba_o.d_source [7]
+Toggle tl_rom_o.a_source [7]
+Toggle tl_rom_i.d_source [7]
+Toggle tl_debug_mem_o.a_source [7]
+Toggle tl_debug_mem_i.d_source [7]
+Toggle tl_ram_main_o.a_source [7]
+Toggle tl_ram_main_i.d_source [7]
+Toggle tl_eflash_o.a_source [7]
+Toggle tl_eflash_i.d_source [7]
+Toggle tl_peri_o.a_source [7]
+Toggle tl_peri_i.d_source [7]
+Toggle tl_flash_ctrl_o.a_source [7]
+Toggle tl_flash_ctrl_i.d_source [7]
+Toggle tl_hmac_o.a_source [7]
+Toggle tl_hmac_i.d_source [7]
+Toggle tl_aes_o.a_source [7]
+Toggle tl_aes_i.d_source [7]
+Toggle tl_rv_plic_o.a_source [7]
+Toggle tl_rv_plic_i.d_source [7]
+Toggle tl_pinmux_o.a_source [7]
+Toggle tl_pinmux_i.d_source [7]
+Toggle tl_padctrl_o.a_source [7]
+Toggle tl_padctrl_i.d_source [7]
+Toggle tl_alert_handler_o.a_source [7]
+Toggle tl_alert_handler_i.d_source [7]
+Toggle tl_nmi_gen_o.a_source [7]
+Toggle tl_nmi_gen_i.d_source [7]
+Toggle tl_otbn_o.a_source [7]
+Toggle tl_otbn_i.d_source [7]
+Toggle tl_keymgr_o.a_source [7]
+Toggle tl_keymgr_i.d_source [7]
+Toggle tl_corei_i.a_source [6]
+Toggle tl_corei_o.d_source [6]
+Toggle tl_cored_i.a_source [6]
+Toggle tl_cored_o.d_source [6]
+Toggle tl_dm_sba_i.a_source [6]
+Toggle tl_dm_sba_o.d_source [6]
+Toggle tl_rom_o.a_source [6]
+Toggle tl_rom_i.d_source [6]
+Toggle tl_debug_mem_o.a_source [6]
+Toggle tl_debug_mem_i.d_source [6]
+Toggle tl_ram_main_o.a_source [6]
+Toggle tl_ram_main_i.d_source [6]
+Toggle tl_eflash_o.a_source [6]
+Toggle tl_eflash_i.d_source [6]
+Toggle tl_peri_o.a_source [6]
+Toggle tl_peri_i.d_source [6]
+Toggle tl_flash_ctrl_o.a_source [6]
+Toggle tl_flash_ctrl_i.d_source [6]
+Toggle tl_hmac_o.a_source [6]
+Toggle tl_hmac_i.d_source [6]
+Toggle tl_aes_o.a_source [6]
+Toggle tl_aes_i.d_source [6]
+Toggle tl_rv_plic_o.a_source [6]
+Toggle tl_rv_plic_i.d_source [6]
+Toggle tl_pinmux_o.a_source [6]
+Toggle tl_pinmux_i.d_source [6]
+Toggle tl_padctrl_o.a_source [6]
+Toggle tl_padctrl_i.d_source [6]
+Toggle tl_alert_handler_o.a_source [6]
+Toggle tl_alert_handler_i.d_source [6]
+Toggle tl_nmi_gen_o.a_source [6]
+Toggle tl_nmi_gen_i.d_source [6]
+Toggle tl_otbn_o.a_source [6]
+Toggle tl_otbn_i.d_source [6]
+Toggle tl_keymgr_o.a_source [6]
+Toggle tl_keymgr_i.d_source [6]
+
+Toggle tl_corei_i.a_param
+Toggle tl_corei_o.d_param
+
+Toggle tl_corei_o.d_user
+Toggle tl_corei_i.a_user.rsvd1
+Toggle tl_corei_i.a_user.parity
+Toggle tl_corei_i.a_user.parity_en
+
+Toggle tl_corei_o.d_opcode [2:1]
```

Comment:
Does range selection work here? If yes then it should work on other fields of the struct as well, right?